src/components/visualizations/headline/utils/test/calculateXirr.spec.ts

```diff
@@ -27,7 +27,7 @@ describe("calculateXirr", () => {
         ];
 
         const actual = calculateXirr(xirrInput);
-        const expected = 0.37336253350955556;
+        const expected = 0.3733625335095555;
```

Comment:
Arithmetics were changed somewhere between four major versions of node.js we are upgrading to. Confirmed with @no23reason that it's OK in this case.